scripts/experiments_tabular/wae.jl

```diff
@@ -0,0 +1,164 @@
+using DrWatson
+@quickactivate
+using ArgParse
+using GenerativeAD
+import StatsBase: fit!, predict
+using StatsBase
+using BSON
+using Flux
+using IPMeasures
+using GenerativeModels
+using Distributions
+
+s = ArgParseSettings()
+@add_arg_table! s begin
+   "max_seed"
+        required = true
+        arg_type = Int
+        help = "seed"
+    "dataset"
+        required = true
+        arg_type = String
+        help = "dataset"
+end
+# for testing purposes
+if length(ARGS) == 0
+	push!(ARGS, "1")
+	push!(ARGS, "iris")
+end
```

Comment:
I would leave this out, even though it does nothing when run in batch with some arguments.

Reply:
I'd like to keep it for cases when I test this in REPL.

Reply:
Also, it is needed in order to avoid errors when including the script elsewhere.